### DISCUSSION TITLE
current cpu load average 1.86 is higher than 1.0 or available memory 0.0045294471387615776 is lower than 0.3

### DISCUSSION THREAD

**Discussion by bongmu** [apache/dolphinscheduler#15047] [Q&A]
deploy with kubernetes
There is no instance after the workflow is clicked to run.
error：
current cpu load average 1.86 is higher than 1.0 or available memory 0.0045294471387615776 is lower than 0.3

How to solve it？
![image](https://github.com/apache/dolphinscheduler/assets/33013978/195a06aa-2808-40a0-b78c-327c9db59e01)

**Comment by bongmu** [comment]
server configuration:
128 core， 512G memory，There is no insufficient server hardware configuration

**Reply by CHINAJAVABoy** [reply]
66666666666666

**Comment by bongmu** [comment]
version：3.2.0
There is no problem with version 3.1.8 and before

**Comment by truonglac2603** [comment]
i'm facing the same issue

**Comment by RaymondFishWang** [comment]
you can change the k8s configuration for 
 - name: MASTER_MAX_CPU_LOAD_AVG
      value: "-1"
    - name: MASTER_RESERVED_MEMORY
      value: "****"
